### ISSUE TITLE
Render block content using the same widget wrappers

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#25670]
There appears to be a missing implementation detail in the way block widgets render their content, as detailed below:

Fig.1 - the old output is rendered with wrappers:

```
<div class="widget widget_archive">
	<div class="widget-content">
<!-- actual widget content -->
		<h2 class="widget-title subheading heading-size-3">Archives</h2>
			<ul>
					...
			</ul>
	</div>
</div>
<div class="widget widget_recent_comments">
...
```

Fig.2 - the new output is rendered without wrappers:

```
<div class="widget widget_archive">
	<div class="widget-content">
...
	</div>
</div>
<!-- actual block content -->
<div class="wp-block-columns">
	<div class="wp-block-column">
		<h2>asdasdasd</h2>
	</div>
	<div class="wp-block-column">
		<figure class="wp-block-image size-large"><img src="http://localhost:8889/wp-content/uploads/2020/09/2020-landscape-1-1024x656.png" alt="" class="wp-image-5"></figure>
	</div>
</div>
```

We need to preserve the wrappers rendered on the front-end so that theme styling kicks in properly.

**Comment by kevin940726** [comment]
I'm not sure I followed the description here. Could you provide reproduction steps?

**Comment by adamziel** [comment]
@kevin940726 this is the HTML structure rendered on the frontend:

<img width="522" alt="Zrzut ekranu 2020-09-29 o 10 41 11" src="https://user-images.githubusercontent.com/205419/94534135-53ade580-0240-11eb-9bc0-2a6278a0888e.png">

All widgets, including blocks, should be wrapped in the same structure. I'm looking into that now.

**Comment by draganescu** [comment]
Closed by #25693
